pypit/arcoadd.py

```diff
@@ -460,66 +461,92 @@ def clean_cr(spectra, smask, n_grow_mask=1, nsig=5., nrej_low=5., debug=False, *
 
     if spectra.nspec == 2:
         msgs.info("Only 2 exposures.  Using custom procedure")
-        '''
-        diff = fluxes[0,:] - fluxes[1,:]*med_scale
-        # Spec0?
-        spectra.select = 0
-        cr0 = diff > nsig*np.sqrt(spectra.data['sig'][0,:]**2 + med_scale**2 *
-            spectra.data['sig'][1,:]**2)
-        if n_grow_mask > 0:
-            cr0 = grow_mask(cr0, n_grow=n_grow_mask)
-        msgs.info("Rejecting {:d} CRs in exposure 0".format(np.sum(cr0)))
-        spectra.add_to_mask(cr0)
-        # Spec1?
-        spectra.select = 1
-        cr1 = (-1*diff) > nsig*np.sqrt(spectra.data['sig'][0,:]**2 + med_scale**2 *
-                                  spectra.data['sig'][1,:]**2)
-        if n_grow_mask > 0:
-            cr1 = grow_mask(cr1, n_grow=n_grow_mask)
-        spectra.add_to_mask(cr1)
-        msgs.info("Rejecting {:d} CRs in exposure 1".format(np.sum(cr1)))
-        '''
-        # b-spline approach
-        # Package Data for convenience
-        waves = spectra.data['wave'].flatten()  # Packed 0,1
-        flux = fluxes.flatten()
-        sig = sigs.flatten()
-        #
-        gd = np.where(sig > 0.)[0]
-        srt = np.argsort(waves[gd])
-        idx = gd[srt]
-        mask, spl = arutils.robust_polyfit(waves[idx], flux[idx], 3, function='bspline',
-                weights=1./sig[gd][srt], sigma=3., maxone=False, everyn=6)
-        # Reject CR (with grow)
-        spec_fit = arutils.func_val(spl, wave, 'bspline')
-        for ii in range(2):
-            diff = fluxes[ii,:] - spec_fit
-            cr = (diff > nsig*sigs[ii,:]) & (sigs[ii,:]>0.)
-            if False:
-                debugger.plot1d(spectra.data['wave'][0,:], spectra.data['flux'][ii,:], spec_fit, xtwo=spectra.data['wave'][0,cr], ytwo=spectra.data['flux'][ii,cr], mtwo='s')
+        if cr_two_alg == 'diff': # b-spline approach
```

Comment:
This should be '# diff approach' instead of '# bspline approach' (sorry, nitpicky comment, but it confused be for a bit).